docs/products/grafana/howto/dashboard-previews.rst

```diff
@@ -0,0 +1,39 @@
+Dashboard preview for Aiven for Grafana®
+========================================
+
+Grafana's Dashboard Previews feature is a great way to get an overview of all your dashboards. Instead of a list view of all your dashboards, you can now see a graphical representation for each dashboard configured.
+
+The dashboard previews feature is an opt-in beta feature available in Grafana 9.0+. This feature is disabled by default on the Aiven for Grafana® service. 
+
+.. note:: 
+    Dashboard previews are not available for Hobbyist and Startup-1 plans.
```

Comment:
this is more like small suggestion but it is up to you. do you think it makes sense to say which plans they are available? rather than the ones that they are not?

Reply:
Well, ya. But if new service plans are added (with higher service plans), they probably will also support this feature, and in that case, we don't have to update the document constantly. I think I will leave this for now. See how the service plans evolve and take a call later

Reply:
ok, seems like a great idea